tipg/database.py

```diff
@@ -61,10 +62,14 @@ async def __call__(self, conn: asyncpg.Connection):
 
         # Register custom SQL functions/table/views in pg_temp
         for sqlfile in self.user_sql_files:
-            await conn.execute(sqlfile.read_text())
+            await conn.execute(
+                sqlfile.read_text().replace("pg_temp", settings.tipg_schema)
```

Comment:
I'm trying to think about the use case for the custom function to have access the the `pg_temp` schema?